hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncTableImpl.java

```diff
@@ -220,7 +220,8 @@ private static Result toResult(HBaseRpcController controller, MutateResponse res
   public CompletableFuture<Result> get(Get get) {
     return timelineConsistentRead(conn.getLocator(), tableName, get, get.getRow(),
       RegionLocateType.CURRENT, replicaId -> get(get, replicaId), readRpcTimeoutNs,
-      conn.connConf.getPrimaryCallTimeoutNs(), retryTimer, conn.getConnectionMetrics());
+      conn.connConf.getPrimaryCallTimeoutNs(), retryTimer,
+      conn.getConnectionMetrics().orElse(null));
```

Comment:
This is what I mean, we unwrap the Optional, and then check if it is null, this does not make sense...